docs/UserGuide.md

```diff
@@ -191,6 +191,22 @@ This represents a company, any company, but typically one you are interested in
 You can record and maintain a company's name, phone number, email address, physical address, and its industry types.
 You can then specify what internships the company is offering by using the [internship commands](#internship).
 
+Note that all Company commands follow the following general 2-step process:
+
+Step 1: Enter the command into the command box. An example command is given in the image below.
+
+![GeneralStep1MarkUp](images/ug-company/GeneralStep1MarkUp.png)
+
+<p align="center">Figure 2: Step 1 for all Company commands, with an example</p>
+
+Step 2: Press 'Enter' on your keyboard to execute the command. Upon successful execution, you will see the results of 
+your command reflected immediately in the app. An appropriate result message will also be displayed in the result 
+display.
+
+![GeneralStep2MarkUp](images/ug-company/GeneralStep2MarkUp.png)
+
+<p align="center">Figure 3: Step 2 for all Company commands, with an example</p>
```

Comment:
I think the figure number and description should be italicized since it is a side information?

Reply:
As per today's discussion, will do away with these captions instead. Thanks for pointing this out though!